test/support/api/forms.js

```diff
@@ -1,62 +1,54 @@
 import _ from 'underscore';
-import { v4 as uuid } from 'uuid';
 import { getResource, mergeJsonApi } from 'helpers/json-api';
 
-import fxTestForms from 'fixtures/test/forms';
+import fxTestForms from 'fixtures/collections/forms';
+
+const fxSampleForms = _.rest(fxTestForms, 2);
 
 const TYPE = 'forms';
 
 export function getForm(data) {
-  const resource = getResource(_.sample(fxTestForms), TYPE);
-
-  data = _.extend({ id: uuid() }, data);
+  const resource = getResource(_.sample(fxSampleForms), TYPE);
 
   return mergeJsonApi(resource, data);
 }
 
 export function getForms() {
-  return getResource(fxTestForms, TYPE);
+  return getResource(fxSampleForms, TYPE);
 }
 
 // Exporting only form needed for testing variance
-export const testForm = getResource(_.find(fxTestForms, { id: '11111' }));
+export const testForm = getResource(_.extend(fxTestForms[0], {
+  name: 'Test Form',
+}), TYPE);
 
 Cypress.Commands.add('routeForms', (mutator = _.identity) => {
-  // form.options is no longer included in the '/api/forms' api request
-  const fxForms = _.map(fxTestForms, form => {
-    form = _.clone(form);
-    form.options = {};
-    return form;
-  });
-
   cy
     .intercept('GET', '/api/forms', {
       body: mutator({
-        data: getResource(fxForms, TYPE),
+        data: getResource(fxTestForms, TYPE),
```

Comment:
I think this should be `data: [testForm, ...getResource(fxSampleForms, TYPE)],`

Reply:
otherwise this the forms url will contain the `testForm.id` with different attributes than loaded elsewhere.

Reply:
Yep, good call, updated 👍